xcode/Subconscious/Shared/Components/Detail/DetailStackView.swift

```diff
@@ -105,14 +105,13 @@ enum DetailStackAction: Hashable {
     /// request an editor detail.
     case findAndPushDetail(
         address: Slashlink,
-        link: SubSlashlinkLink? = nil
+        fallback: String?
     )
     
     case findAndPushLinkDetail(
-        context: ResolvedAddress,
+        context: Peer?,
         link: SubSlashlinkLink
     )
```

Comment:
I'm slightly firming up the difference between these cases.

`findAndPushDetail` is for taking an address and pushing the correct navigation item onto the stack for it. It takes an optional `fallback` parameter now for the editor case.

`findAndPushLinkDetail` will find the best address for a link, then call through to `findAndPushDetail`. It should populate `fallback` if possible.